libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/MapboxCopilotImpl.kt

```diff
@@ -109,15 +110,20 @@ internal class MapboxCopilotImpl(
     }
     private var initRoute = false
     private lateinit var routesObserver: RoutesObserver
+    private val isMapboxNavigationAppSetup = MapboxNavigationApp.isSetup()
```

Comment:
@Zayankovsky and I were discussing this internally
> It would be weird if they setup `MapboxNavigationApp` after `MapboxNavigation` was manually created. I think this will even crash.